lib/stat_tracker.rb

```diff
@@ -456,4 +456,20 @@ def fewest_tackles(season_id)
     team_names[tackle_stats.key(tackle_stats.values.min)]
   end
 
+  def team_info(team_id)
+    teams = CSV.open(@team_path, headers: true, header_converters: :symbol)
+    team_hash = Hash.new()
+    teams.each do |row|
+      if row[:team_id] == team_id
```

Comment:
there may be a way to just return the whole row as a hash without the stadium instead of creating a new hash and reassigning values. but i need to research more on this